### PR TITLE
Wconstab/maskrcnn2

### PR DESCRIPTION
**Pull request by wconstab**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,6 +13,9 @@ jobs:
       - run:
           name: Install Conda
           command: ./scripts/install_basics.sh
+      - run:
```
Comment (zdevito) [inline .circleci/config.yml:16]: Why did this become necessary? Hitting the nvidia servers every time will make this more flaky, right?

**Review by zdevito — APPROVED** [review]
Thanks!

File: .circleci/config.yml (hunk 1 above)
Comment (wconstab) [reply .circleci/config.yml:16]: i needed nvcc.

ultimately i disabled building/using apex/amp due to it introducing a memory leak (and it took a while to compile too annoyingly)

forgot if anything else in the repo needed nvcc but i kind of think so.

Comment (wconstab) [reply .circleci/config.yml:16]: not sure how much to worry about hitting the nv servers; but i'd be up for an alternative way if you have a suggestion.  this script/logic is copied from somewhere in pytorch repo, i thought it was already being used for those CI builds.

**Pull request merged**